postcss.config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    plugins: [
+      require('postcss-import'),
+      require('tailwindcss/nesting'),
+      require('tailwindcss'),
+      require('autoprefixer'),
+      require("postcss-flexbugs-fixes"),
+      require("postcss-preset-env")({
+        autoprefixer: {
+          flexbox: "no-2009"
```

Comment:
can't autoprefixer use the browserslist?

Reply:
I took this from the Dodona postcss.config.js, so I assumed there is a good reason why it is present (as this is supposed to work with Dodona, so the same config should help).

Reply:
I don't think we actually use postcss in Dodona (it might have been added by webpacker, but all css is actually handled by sprockets)

Reply:
https://github.com/dodona-edu/dodona/blob/develop/postcss.config.js is indeed a rather old file. Which parts of this config file do you think are irrelevant then?

Reply:
All of them. No settings are needed. By default the browserslist file is used

Reply:
Removed it then, thank you!